.github/workflows/e2e-win.yml

```diff
@@ -33,11 +33,21 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
           registry-url: https://registry.npmjs.org/ # Needed for auth
+      - name: cache all node_modules
+        id: cache-modules
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          # This is intentionally set to Linux, since Windows produces broken node_modules
+          # cache tarballs because it follows symlinks.
+          key: Linux-node_modules-${{ hashFiles('yarn.lock', '**/package.json') }}
```

Comment:
But why is this in the `e2e-win` file ? 🙃

Reply:
see the comment above :)

Reply:
I tried 😅